packages/backend/src/accounting/psql/service.ts

```diff
@@ -286,9 +293,9 @@ async function createAccountWithdrawal(
 
 async function voidAccountWithdrawal(
   deps: ServiceDependencies,
-  withdrawalId: string
+  transferRef: string
 ): Promise<void | TransferError> {
-  throw new Error('Not implemented')
+  return await voidTransfer(deps, transferRef)
```

Comment:
could probably just have the service call directly, but kept it for consistency?